lib/trento_web/controllers/cluster_controller.ex

```diff
@@ -6,6 +6,8 @@ defmodule TrentoWeb.ClusterController do
     Tags
   }
 
+  alias Trento.Integration.Checks
```

Comment:
Should maybe this go in a new `ChecksController` file?

Reply:
#259